src/content/fr/2019/caching.md

```diff
@@ -0,0 +1,691 @@
+
+Les architectures Web impliquent généralement [une mise en cache en plusieurs niveaux](https://blog.yoav.ws/tale-of-four-caches/). Par exemple une requête HTTP peut être mise en cache de différentes manière&nbsp;:
+
+*   dans le cache du navigateur&nbsp;;
+*   dans le cache d'un <i lang="en">service worker</i> dans le navigateur&nbsp;;
+*   dans une passerelle partagée&nbsp;;
+*   au niveau des CDN, qui offrent la possibilité de mettre en cache à proximité des utilisateurs&nbsp;;
+*   dans un proxy de cache en amont des applications pour réduire la charge sur les serveurs back-end&nbsp;;
+*   au niveau de l'application et de la base de données.
+
+Ce chapitre explique comment les ressources sont mises en cache dans les navigateurs Web.
+
+## Présentation de la mise en cache HTTP
+
+Pour qu'un client HTTP mette en cache une ressource, il doit répondre a deux questions&nbsp;:
+
+*   "Combien de temps dois-je mettre en cache&nbsp;?"
+*   "Comment puis-je valider que le contenu est encore frais&nbsp;?"
+
+Lorsqu'un navigateur Web envoie une réponse à un client, il inclut généralement dans sa réponse des en-têtes qui indiquent si la ressource peut être mise en cache, pour combien de temps et quel est son âge. La RFC 7234 traite plus en détail de ce point dans la section [4.2 (Freshness)](https://tools.ietf.org/html/rfc7234#section-4.2) et [4.3 (Validation)](https://tools.ietf.org/html/rfc7234#section-4.3).
+
+Les en-têtes de réponse HTTP généralement utilisées pour transmettre la durée de vie sont&nbsp;:
+
+*   `Cache-Control` vous permet de configurer la durée de vie du cache (c'est-à-dire sa durée de validité).
+*   `Expires` fournit une date ou une heure d'expiration (c.-à-d. quand exactement celle-ci expire).
+
+`Cache-Control` est prioritaire si les deux champs sont renseignés. Ces en-têtes sont [abordés plus en détail ci-dessous](#cache-control-vs-expires).
+
+Les en-têtes de réponse HTTP permettant de valider les données stockées en cache, c'est à dire donner les informations nécessaires pour comparer une ressource à sa contrepartie côté serveur&nbsp;:
+
+*   `Last-Modified` indique quand la ressource a été modifiée pour la dernière fois.
+*   `ETag` fournit l'identifiant unique d'une ressource.
+
+`ETag` est prioritaire si les deux en-têtes sont renseignés. Ces en-têtes sont [abordés plus en détail ci-dessous](#validating-freshness).
```

Comment:
Since you've translated this header, I think you will need to change the link to the translated version:

```suggestion
`ETag` est prioritaire si les deux en-têtes sont renseignés. Ces en-têtes sont [abordés plus en détail ci-dessous](#validation-de-la-fraîcheur-des-informations).
```